src/vm/arm/cgencpu.h

```diff
@@ -15,6 +15,8 @@
 #include "utilcode.h"
 #include "tls.h"
 
+typedef BYTE byte;
```

Comment:
Looking at the source, I can see that the only usage of lowercase "byte" in this file is in this line:
EXTERN_C void FCallMemcpy(byte* dest, byte* src, int len);
I would modify it to use the uppercase BYTE.